### DISCUSSION TITLE
Autowax after new update

### DISCUSSION THREAD

**Discussion by chuuiy** [thatskymod/Sky-CotL-Scripts#70] [General]
Everytime I use autowax, it always says current session was terminated. Even if I reload it won't work.

**Comment by thatskymod** [comment]
Thank you for reaching out, @chuuiy.

The session error is now resolved in AW Dukun i-Sky v1.5.0.

An updated file is available for download at https://github.com/thatskymod/Sky-CotL-Scripts/blob/main/Canvas/Libs/libdukun150.so.
